packages/node/package.json

```diff
@@ -43,19 +43,23 @@
   },
   "dependencies": {
     "@novu/shared": "^0.20.0-alpha.0",
+    "axios-retry": "^3.8.0",
     "handlebars": "^4.7.7",
     "lodash.get": "^4.4.2",
-    "lodash.merge": "^4.6.2"
+    "lodash.merge": "^4.6.2",
+    "uuid": "^9.0.1"
```

Comment:
@michaldziuba03 `UUID` is already part of node internals. You can just import `import { randomUUID } from 'node:crypto'` instead of adding a new package

Reply:
yes, I know but not sure about the support of `randomUUID` in older Node.js versions.

https://nodejs.org/api/crypto.html#cryptorandomuuidoptions

It was added in `v15.6.0, v14.17.0` and in `package.json` technically SDK supports Node.js 10+
```json
  "engines": {
    "node": ">=10"
  },
```

Reply:
Oh wow. I think Novu maintainers need to bump the node version to 14 at least for the package.

Reply:
...And drop support for node 10